### ISSUE TITLE
Repodata patches - staging server

### ISSUE THREAD

**Issue by wolfv** [mamba-org/quetz#126]
Currently applying repodata patches is hard to test. It would be cool to have a plugin to stage repodata patches so that one can run some tests against the patches, before rolling them out to everyone.

**Comment by btel** [comment]
note that the as of #130 repodata patching is now implemented as quetz-repodata_patching plugin

**Comment by wolfv** [comment]
yep! One other thought on this would be to have a third mirroring option that would only apply the repodata patch, but not host any packages... so it would be a fully virtual channel.

That way we could test new repodata patches for e.g. conda-forge and see if they work fine by pointing to `conda-forge-staging` or something like that.

**Comment by adriendelsalle** [comment]
> that would only apply the repodata patch, but not host any packages...  so it would be a fully virtual channel.

@wolfv Same technique as required for #53 ?
